kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java

```diff
@@ -515,7 +509,7 @@ private ContextMenu buildMenuOptionContextMenu() {
         Object[][] menuItems;
         // show the 'Add Fully Qualified' option when it is a new concept in create mode and there is no fully qualified name
         if (this.conceptViewModel.getPropertyValue(MODE).equals(CREATE) &&
-            (getConceptViewModel().getPropertyValue(FULLY_QUALIFIED_NAME) == null)) {
+                (getConceptViewModel().getPropertyValue(FULLY_QUALIFIED_NAME) == null)) {
```

Comment:
**NOTE:** I may have to fix(revisit) Cognitive's `getProperty(...)` method to not return type `T` but instead `Property<T>`.
That way you could do the following:
```java
 .getProperty("someProp").isNull()
```

Reply:
@indritbeqiri  Nothing you need to do. I just wanted to point that out. My intent is to improve the mvvm api.

Reply:
Thanks for pointing that out, Carl. We totally agree that switching to `Property<T>` would make the MVVM API more consistent and flexible.

@besidev @indritbeqiri